client/src/pages/Translations.jsx

```diff
@@ -1,17 +1,112 @@
 // @flow
 
-import React from 'react';
+import React, { useState, useEffect, useCallback } from 'react';
 import { Tabs, Layout, Row, Table } from 'antd';
 import { Link } from 'react-router-dom';
 import '../css/Translations.css';
 import StatusTag from '../components/StatusTag';
 import PriorityIcon from '../components/PriorityIcon';
+import { getVerifications } from '../utils/api';
 
 const { Header } = Layout;
 const { TabPane } = Tabs;
 
+const languages = ['Spanish', 'French', 'Chinese'];
+const priorities = { Urgent: 4, High: 3, Medium: 2, Low: 1 };
+
 function Translations() {
-  const columns = [
+  const [verifiedList, setVerifiedList] = useState({});
+  const [unverifiedList, setUnverifiedList] = useState({});
+
+  const updateLists = useCallback((verifications) => {
+    const verifiedResources = { Spanish: [], French: [], Chinese: [] };
+    const unverifiedResources = { Spanish: [], French: [], Chinese: [] };
+    languages.forEach((l) => {
+      const resources = verifications[l];
+
+      resources?.forEach((verification) => {
+        const percentage =
+          (verification.verifiedTranslationsCount /
+            verification.totalTranslations) *
+          100;
+
+        if (verification.totalReports >= 1) {
+          unverifiedResources[l].push({
+            key: '1',
+            name: verification.name,
+            priority: <PriorityIcon priorityType="Urgent" />,
+            status: <StatusTag status="Unverified" />,
+            translate: (
+              <Link
+                to={
+                  `/translation/${verification._id}?` +
+                  `language=${l}&type=${verification.type}`
+                }
+              >
+                Translate
+              </Link>
+            ),
+          });
+        } else if (percentage === 100.0) {
+          verifiedResources[l].push({
+            key: '1',
```

Comment:
In general I think keys should be unique - let's make all of the keys `verification._id`